WordPress/src/main/java/org/wordpress/android/push/NotificationsProcessingService.java

```diff
@@ -596,6 +618,12 @@ private void resetOriginalNotification() {
         }
     }
 
+    private void trackQuickActionCompleted(QuickActionTrackPropertyValue type) {
+        Map<String, String> properties = new HashMap<>(1);
+        properties.put("quick_action", type.toString());
+        AnalyticsTracker.track(AnalyticsTracker.Stat.NOTIFICATION_QUICK_ACTIONS_QUICKACTION_TOUCH, properties);
```

Comment:
I would add a call to ` AnalyticsTracker.flush();` here - To be sure all the Analytics data are sent to the remote server.

Reply:
👍 good call, added in 55a0d7c063